data/csp/oci/sle15/packages.yaml

```diff
@@ -0,0 +1,13 @@
+packages:
+  image:
+    OCI:
+      - firewalld
+      - libselinux1
+      - open-iscsi
+      - selinux-tools
+      - cloud-init
+drivers:
```

Comment:
Not sure if this is how to handle the drivers?

Reply:
Format would be:
```
drivers:
  _namespace_foo:
    file:
      - _attributes:
          name: ixgbe
      ...
```
Which then can be included in a `drivers` section in the image definition.